tests/e2e/specs/cache-control-headers-directives.test.js

```diff
@@ -0,0 +1,36 @@
+import {
+	visitAdminPage,
+	createNewPost,
+	trashAllPosts,
+	createURL,
+	logout,
+} from "@wordpress/e2e-test-utils";
+
+describe( 'Cache Control header directives', () => {
+
+	beforeEach( async () => {
+		await trashAllPosts();
+	} );
+
+	it( 'No private directive present in cache control when user not logged in.', async () => {
+		await createNewPost( {
+			title: 'Hello World',
+			post_status: 'publish',
+		} );
+
+		await logout();
+		const response = await page.goto( createURL( '/hello-world/' ) );
+
+		const cacheControl = response.headers();
+		expect( cacheControl[ 'cache-control' ] ).not.toContain( 'private' );
+	} );
+
+	it( 'Private directive header present in cache control when logged in.', async () => {
+		await visitAdminPage( '/wp-admin' );
+
+		const response = await page.goto( createURL( '/wp-admin' ) );
+		const cacheControl = response.headers();
+		expect( cacheControl[ 'cache-control' ] ).toContain( 'private' );
+	} );
+
+} );
```

Comment:
```suggestion
} );

```

Add empty line at the end.